identification-bff.yml

```diff
@@ -208,22 +246,57 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Customer'
-  /api/v1/identification-bff/vendors:
+  /api/v1/identification-bff/vendors?pageNo=[pageNo]&pageSize=[pageSize]&firstName=[firstName]&lastName=[lastName]&userName=[userName]&legalName=[legalName]&commonName=[commonName]&status=[status]&address=[address]:
     get:
       tags:
         - vendor-handler
       operationId: getVendors
       parameters:
         - in: query
-          name: page
+          name: pageNo
           required: false
           schema:
             type: integer
         - in: query
-          name: size
+          name: pageSize
           required: false
           schema:
             type: integer
+        - in: query
+          name: firstName
+          required: false
+          schema:
+            type: string
+        - in: query
+          name: lastName
+          required: false
+          schema:
+            type: string
+        - in: query
+          name: userName
+          required: false
+          schema:
+            type: string
+        - in: query
```

Comment:
I thought we that would be one field searching if 2 columns in the database. It could be something like business name.